src/main/jbake/content/ejb-intro004.adoc

```diff
@@ -70,7 +70,7 @@ and Directory Interface syntax to find the enterprise bean instance.
 
 Dependency injection is the simplest way of obtaining an enterprise bean
 reference. Clients that run within a Jakarta EE server-managed environment,
-JavaServer Faces web applications, JAX-RS web services, other enterprise
+Jakarta Server Faces web applications, Jakarta RESTful web services, other enterprise
```

Comment:
I didn't capitalize "Web Services" on purpose. Hopefully we'll be able to use "Jakarta REST" as short name, but I'm still not sure about it.